batch_import/samples/sample_hook_data.json

```diff
@@ -1,6 +1,8021 @@
 {
-    "plan": "pro",
-    "img_y_size_extraction": 1080,
-    "img_x_size_cover": 1920,
-    "img_y_size_cover": 1080
-}
+  "labels": {
```

Comment:
@jhallard something's wrong since this file is now output rather than the intended hook data.